ui/src/pages/version/components/forms/components/ImageBuilderSection.js

```diff
@@ -1,64 +1,72 @@
 import { FormLabelWithToolTip, useOnChangeHandler } from "@caraml-dev/ui-lib";
-import { 
-    EuiFieldText, 
-    EuiFlexGroup, 
-    EuiFlexItem, 
-    EuiFormRow, 
+import {
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiTitle,
 } from "@elastic/eui";
 
 
 export const ImageBuilderSection = ({
-    imageBuilderResourceConfig,
-    onChangeHandler,
-    errors = {},
-  }) => {
+  imageBuilderResourceConfig,
+  onChangeHandler,
+  errors = {},
+}) => {
+  const {onChange} = useOnChangeHandler(onChangeHandler);
 
-    const { onChange } = useOnChangeHandler(onChangeHandler);
-
-    return (
+  return (
+    <>
+      <EuiTitle size="xs">
+        <h4>{"Image Building Resources"}</h4>
+      </EuiTitle>
```

Comment:
I moved the title into this component (the title used to be in the parent component).